src/iotjs_binding.cpp

```diff
@@ -139,142 +115,126 @@ JObject::~JObject() {
 }
 
 
-static JRawValueType jnull_val = JVal::Null();
-static JObject jnull(&jnull_val);
-JObject& JObject::Null() {
-  return jnull;
+JObject JObject::Null() {
+  return JObject(JVal::Null(), false);
```

Comment:
jerry api is not available before the jerry_init() and static initializers run at startup, before the jerry_init(), so we cannot declare it as a static.